src/components/SettingsAdvanced.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+    <div class="kiwi-settings-advanced">
+        <div class="kiwi-settings-advanced-notice">{{ $t('settings_advanced_header') }}</div>
+        <form class="u-form">
+            <div class="kiwi-settings-advanced-filter">
+                <div class="kiwi-settings-filter-container">
+                    <input id="filter" v-model="filterString"
+                           :placeholder="$t('settings_advanced_filter')"
+                           class="u-input">
+                    <i v-if="!filterString" class="fa fa-search" aria-hidden="true"/>
+                    <i v-else class="fa fa-times" aria-hidden="true" @click="filterString = ''"/>
+                </div>
+            </div>
+            <table class="u-table kiwi-settings-advanced-table" cellspacing="0">
+                <p v-if="filteredSettings.length === 0" class="kiwi-settings-advanced-noresult">
```

Comment:
A `<p>` element cannot be a child of a `<table>`.  This needs to be wrapped in a `<tr><td>` set of elements.